src/Microsoft.AspNetCore.Mvc.TagHelpers/Resources.resx

```diff
@@ -147,4 +147,16 @@
   <data name="PropertyOfTypeCannotBeNull" xml:space="preserve">
     <value>The '{0}' property of '{1}' must not be null.</value>
   </data>
+  <data name="ButtonTagHelper_CannotOverrideFormAction" xml:space="preserve">
```

Comment:
:see_no_evil: Never mind. "a" versus "an" and the different ordering of the `FormTagHelper_CannotOverrideAction` parameters makes this difficult.